plugins/arangodbendorsements/main_test.go

```diff
@@ -18,15 +18,18 @@ import (
 
```

Comment:
Just a general comment, I assume the Arango DB Unit tests still are passing, with this change ? Kindly confirm and resolve this comment!